### ISSUE TITLE
Failed build

### ISSUE THREAD

**Issue by thinkerytim** [akeeba/akeebasubs#142]
Using akeebasubs branch `4.x`, fof branch `fof3-wip`, results in this when I run `phing git`:

    ~build git:(4.x) phing git
	Buildfile: /Users/thinkery/Code/akeebasubs/build/build.xml
	 [property] Loading /Users/thinkery/Code/akeebasubs/build/./build.properties
	 [property] Unable to find property file: /Users/thinkery/Code/akeebasubs/build/./build.properties... skipped

	AkeebaSubscriptions > new-release:

	   [delete] Deleting directory /Users/thinkery/Code/akeebasubs/release
	    [mkdir] Created dir: /Users/thinkery/Code/akeebasubs/release

	AkeebaSubscriptions > setup-properties:

	AkeebaSubscriptions > component-package:

	     [copy] Copying 1 file to /Users/thinkery/Code/akeebasubs/component
	     [copy] Copying 1 file to /Users/thinkery/Code/akeebasubs/component
	     [copy] Copying 1 file to /Users/thinkery/Code/akeebasubs/component/backend
	    [zipme] Building ZIP: /Users/thinkery/Code/akeebasubs/release/com_akeebasubs-revBCE44AB.zip
	bool(false)
	bool(false)
	bool(true)
	bool(false)
	MISSING /Users/thinkery/Code/akeebasubs/component/strapper
	This should never happen!

**Comment by nikosdion** [comment]
Eh, you are mixing things you can't mix. The Akeeba Subs 4.x branch is supposed to be used with FOF 2.x which is currently in the FOF development branch. The Akeeba Subs development branch is supposed to be used with FOF 3.0 which is currently in the FOF fof3-wip branch.

Trying to use Akeeba Subs 4.x (a FOF 2 component) with the backwards incompatible FOF 3 library does cause a spot of a bother, indeed ;)

**Comment by thinkerytim** [comment]
Thanks Nick! So what is the recommended branch of Akeebasubs to use with a Joomla 3.4.x site? Is it the 4.x branch?

**Comment by nikosdion** [comment]
The stable release is the 4.x branch. If you want to provide something stable and reliable to your clients use the 4.x branch. Use it with FOF's development branch.

The in-development Akeeba Subs release is in the development branch. If you want to help me test Akeeba Subs 5 before I release it please use it with FOF's fof3-wip branch (remember to symlink fof3-wip/fof to <site>/libraries/fof30). I have a woefully outdated version of Intellectual Property so I'm not even remotely confident that my integration with your component still works :s

**Comment by thinkerytim** [comment]
Thanks again Nick-- I assumed that the Akeebasubs 4.x would be using the latest fof as well. Is Akeebasubs 5 meant for the current Joomla builds or for pre-release builds? I'd be glad to try it out if I can help you.

I'm also happy to send you a current IProperty build but it seems like since you're no longer supporting the Akeebasubs component you probably aren't interested in updating that. We still do have quite a few users who want to use Akeebasubs to manage their subscriptions but they typically aren't up to building it themselves.

Thanks as always for your ridiculously fast help!

**Comment by nikosdion** [comment]
Akeeba Subs 4 was the last release using FOF 2. It's powering our site as well. If it weren't for the (insert long string of expletives) EU VAT MOSS affair I'd have started working on AS5 much earlier. The AS4 codebase was in much better shape than AS3 but it's still a series of temporary solutions held together by string and duct tape.

Akeeba Subs 5 is currently pre-release. It's rewritten on top of FOF 3. It was both a good chance for me to verify the new version of the framework with a real world application _and_ have an "example" for FOF 3 that goes far beyond the basics. It was also a great opportunity to refactor that haunted codebase which accumulated hacks from the transition from a regular component to a Nooku component to a FOF 1 component to a FOF 2 component. Argh! Believe me, refactoring it was mostly about *removing* code and rewriting some parts which made little to no sense. There's still one ridiculously large model (the Subscribes front-end model) which I haven't touched. Let's just say that I still value my sanity too much to refactor _that_.

Now, regarding Akeeba Subscriptions support status, I've dropped support to end users but not its development! I keep on developing it. It's an integral part of our site's workflow and this won't change, thanks to the EU insanity and the ever changing invoicing and sales regulations. I digress. Integration with IProperty is one of the very few third party integrations I kept, along with those for SOBI and K2. These integrations are what the majority of people were using, they require minimal maintenance and I'm interested in keeping them!

When Akeeba Subs 5 is complete there will be a pre-built package of it on our site that you clients can use. I place that around the end of May 2015. Of course it will come without support from us, but we still have the detailed documentation where they can find the answer to pretty much everything.
